crates/bevy_app/src/plugin_group.rs

```diff
@@ -11,13 +11,19 @@ struct PluginEntry {
     enabled: bool,
 }
 
-#[derive(Default)]
```

Comment:
I don't particularly see a reason for removing Default.
There are a few things to consider
1) Yes it is a smaller code size since the function is not inlined, however this theoretically can make the code slower.
2) This also causes a CI failure, which recommends we have default implementaiton.

Reply:
Performance doesn't matter much as this is startup code only and the difference would be small and likely non-existent with LTO enabled. The saved code size is pretty small though and fixed, so I dropped it.